infra/image/build.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash -eu
+
```

Comment:
not for automated scenario, but I think this script could be also used locally, what about do some additional checks?

# Ensure podman and ansible-playbook commands are available
command -v podman >/dev/null 2>&1 || { echo "ERROR: podman is not installed or not in PATH"; exit 1; }
command -v ansible-playbook >/dev/null 2>&1 || { echo "ERROR: ansible-playbook is not installed or not in PATH"; exit 1; }

Reply:
The check for ansible-playbook is already there. I added the one for podman with the latest commit.